Client/Frontend/Browser/Browser.swift

```diff
@@ -6,7 +6,6 @@ import Foundation
 import WebKit
 
 protocol BrowserHelper {
-    init?(browser: Browser)
```

Comment:
Does it need to be defined on the protocol? I couldn't see anywhere where we had a protocol object we were initializing.